app/client/models/results.js

```diff
@@ -22,7 +22,7 @@ module.exports = (state, bus) => {
   }
 
   const checkfiles = p => p.filesPresent(
-    (err, progress, updated) => { if (updated) render() }
+    (err, progress, updated) => { if (!err && updated) render() }
```

Comment:
Not really handling the error, but standard complains if `err` is not referenced.